sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/DeviceIdentityLifecycleSamples.cs

```diff
@@ -42,6 +42,9 @@ public async Task RunSampleAsync()
             // Update Device Twin.
             await UpdateDeviceTwinAsync(deviceId);
 
+            // Invoke method on device
+            await InvokeMethodOnDeviceAsync(deviceId);
```

Comment:
Adding this block here may make it hard to run this sample. Since you are invoking a method on a newly created device, the user will have to start a device client process mid-sample to make this work.

Reply:
I'd just have a separate sample that takes a device id and invokes a method on that device

Reply:
Do our samples include the device client part of the end-to-end flow, or do specify in comments that a separate application needs to be run in order to call this API successfully?

Reply:
Sounds fair. Will this fail though if the sample is run? I initially thought of creating a separate sample. But it involves creating the device and deleting the device. It just felt like I was repeating the same code again. Things are not really working right now for me to test but if the Invoke method call is just fire and forget, it will be a no-op till user starts the device client. What do you think?

Reply:
It will likely throw a device-not-found exception, if your device isn't online when you invoke the method on it.

Reply:
Ok in that case I think a different sample is better. I'll update the code.